libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -75,6 +75,7 @@
 #define HAL_BOARD_SUBTYPE_CHIBIOS_FMUV5         5013
 #define HAL_BOARD_SUBTYPE_CHIBIOS_MATEKF405WING 5014
 #define HAL_BOARD_SUBTYPE_CHIBIOS_FMUV4PRO      5015
+#define HAL_BOARD_SUBTYPE_CHIBIOS_OMNIBUSF4V6   5016
```

Comment:
why add this? It is not needed unless the board needs special handling somewhere in the code. Most boards don't need that